### PR TITLE
create github action workflow and add IsDir() test case

### PR DESCRIPTION
**Pull request by xiantang**
try to fix https://github.com/cosmtrek/air/issues/87

### CODE REVIEW

**Comment by xiantang** [comment]
try to fix https://github.com/cosmtrek/air/issues/87

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        go-version: ^1.13
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Get dependencies
+      run: |
+        go get -v -t -d ./...
+        if [ -f Gopkg.toml ]; then
```
Comment (cosmtrek) [inline .github/workflows/go.yml:28]: Air is using `go mod` to manage dependencies, so it's unnecessary to check if using dep.

File: .github/workflows/go.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.13
```
Comment (cosmtrek) [inline .github/workflows/go.yml:19]: Can we use go 1.14?

**Comment by xiantang** [comment]
@cosmtrek  we need add required must 1 reviewer rule in some create a pull request.

File: .github/workflows/go.yml (hunk 2 above)
Comment (xiantang) [reply .github/workflows/go.yml]: we can!

File: .github/workflows/go.yml (hunk 1 above)
Comment (xiantang) [reply .github/workflows/go.yml]: done

**Pull request merged**